examples/platforms/nrf52840/platform-config.h

```diff
@@ -431,6 +431,147 @@
 #define OPENTHREAD_PLATFORM_USE_PSEUDO_RESET USB_CDC_AS_SERIAL_TRANSPORT
 #endif
 
+/**
+ * @def Enable power USB detection.
+ *
+ * Configure if the example supports USB port connection.
+ */
+#ifndef USBD_POWER_DETECTION
+#define USBD_POWER_DETECTION 1
+#endif
+
+/**
+ * @def The USB interface to use for CDC ACM COMM.
+ *
+ * According to the USB Specification, interface numbers cannot have gaps. Tailor this value to adhere to this
+ * limitation. Takes values between 0-255
+ */
+#ifndef CDC_ACM_COMM_INTERFACE
+#define CDC_ACM_COMM_INTERFACE 1
```

Comment:
Please prefix this define and one below with `USB_` and move them closer to `USB_CDC_AS_SERIAL_TRANSPORT` define (move them close to each other).